### PR TITLE
adding feature github sync

### PR DESCRIPTION
**Pull request by ColinKotar**
Fetches data from Github API to populate a database as described in #87

### CODE REVIEW

File: gitkanban/gitkanban.py [hunk 1]
```diff
@@ -115,6 +116,10 @@ def define_subcommands(self, subcommands):
         )
         snooze_cmd.set_defaults(func=self.snooze)
 
+        # sync sub-command
+        sync_cmd = SyncCommand(self)
+        sync_cmd.register(subcommands)
+
```
Comment (prashanthellina) [inline gitkanban/gitkanban.py]: @Rajendra-R Let us follow this structure to move out the code for sub-commands from this while wherever it makes sense.

**Comment by prashanthellina** [comment]
@Rajendra-R Please review and post your thoughts.

**Comment by ColinKotar** [comment]
> @ColinKotar Looks good. Please track a separate issue to handle Github API call failures including Rate limits correctly.

Issue #73

File: gitkanban/gitkanban.py (hunk 1 above)
Comment (Rajendra-R) [reply gitkanban/gitkanban.py]: Sure.

File: gitkanban/models.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+    id = Column(Integer, primary_key=True)
+    node_id = Column(String(128))
+
+    created_at = DateTime()
+    updated_at = DateTime()
+
+
+class Organization(BaseMixin, Base):
+    # https://api.github.com/orgs/deep-compute
```
Comment (Rajendra-R) [inline gitkanban/models.py]: @ColinKotar We should not specify company info because it is a public repository.

Comment (prashanthellina) [reply gitkanban/models.py]: @Rajendra-R This reference is alright as this is a publicly accessible API anyway and references the author org of the repo.

**Pull request merged**